checker/src/org/checkerframework/checker/units/Java8TemporalTime.astub

```diff
@@ -0,0 +1,427 @@
+import org.checkerframework.checker.units.qual.*;
+import org.checkerframework.checker.units.qual.time.duration.*;
+import org.checkerframework.checker.units.qual.time.instant.*;
+
+package java.time.temporal;
+
+// enum
```

Comment:
What does the `enum` comment mean? If the stub parser doesn't support enums, is there an issue for this? Reference the issue here, so that it gets fixed.